sw/device/lib/flash_ctrl.h

```diff
@@ -96,6 +96,11 @@ int flash_read(uint32_t addr, part_type_t part, uint32_t size, uint32_t *data);
  */
 void flash_cfg_bank_erase(bank_index_t bank, bool erase_en);
 
+/**
+ * Configure scramble enable
+ */
+void flash_cfg_scramble_en(bool en);
```

Comment:
`en` -> `enable`.

Reply:
sorry i can't tell if this is referring to the function name or the variable. 
I'm guessing function name?

Reply:
As Sam points out, this doesn't Actually Matter. Up to you. =)